tests/commands/bootc.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2025 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use, modify,
+# copy, or redistribute it subject to the terms and conditions of the GNU
+# General Public License v.2.  This program is distributed in the hope that it
+# will be useful, but WITHOUT ANY WARRANTY expressed or implied, including the
+# implied warranties of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.  Any Red Hat
+# trademarks that are incorporated in the source code or documentation are not
+# subject to the GNU General Public License and may only be used or replicated
+# with the express permission of Red Hat, Inc.
+#
+
+import unittest
+from tests.baseclass import CommandTest, CommandSequenceTest
+from pykickstart.commands.bootc import F43_Bootc
+from pykickstart.version import F43
+
+class Bootc_TestCase(unittest.TestCase):
+    def runTest(self):
+        cmd = F43_Bootc()
+
+        # Test arguments that are required
+        op = cmd._getParser()
+        for action in op._actions:
+            if '--source-imgref' in action.option_strings:
+                self.assertEqual(action.required, True)
+
+class F43_TestCase(CommandTest):
+    command = "bootc"
+
+    def runTest(self):
+        # PASS tests
+        cmdstr = "bootc --source-imgref=quay.io/centos-bootc/centos-bootc:stream9"
```

Comment:
When writing PASS tests you should also test the output to make sure it is what you expect. In this case the output is different because you are always filling in the target if it isn't set. So it needs to:
* test the output string
* test with setting a different `--target-imgref`

Reply:
I have added test you required but from some reason it is failing :) I am wondering if this is some string formatting bug.

Reply:
Typo in the assert :) `self.assert_parse(cmdstr, cmdstr)` instead of `self.assert_parse(cmdstr, cmdstr_expected)`
Also still needs a test for using different source and target to demonstrate it not getting copied.

Reply:
Thanks for pointing this out, I have updated the code following you suggestions.